### ISSUE TITLE
Share to e-mail link isn't working

### ISSUE THREAD

**Issue by polaris20** [owncloud/core#18967]
Hello, on both my servers, running PHP 5.5, Apache, and Ubuntu 14.04LTS, and Owncloud 8.1.1, I am not able to share to an e-mail address directly from within the web interface. However, sending a test e-mail from the admin console does work. This function worked fine overall, prior to 8.1.1 (8.0.4). 

Let me know if there's anything I can do to to help troubleshoot. Thanks!

**Comment by oparoz** [comment]
Browser and owncloud logs would help, as well as the list of enabled apps, etc.

**Comment by RealRancor** [comment]
*hüstel* issue template *hüstel*

**Comment by polaris20** [comment]
Enabled Apps:

Activity
Deleted Files
External Sites
External User Support
First Run Wizard
PDF Viewer
Pictures
Share Files
Text Editor
Versions
Video Viewer
Files Move
Imprint
SuperLog

I'll get logs as well.

**Comment by RealRancor** [comment]
@polaris20 Please use the issue template:

https://raw.githubusercontent.com/owncloud/core/master/issue_template.md

to have all needed infos at one place.

**Comment by polaris20** [comment]
Apologies guys; not a ownCloud issue at all. Turns out, there was something weird that occurred between the cloud server and our own internal smtp server (Exchange). Messages were being sent via port 24, however despite no changes in ownCloud nor Exchange, it just stopped working. Changed it temporarily to my Gmail account using their smtp, works fine. Then changed it back to our smtp, but used 25 instead, and voila; it works. 
FYI, I've downloaded the bug report template, so in the future I promise to be a good user, and use that! ;) 

Thanks guys.
